packages/sanity/src/router/IntentLink.tsx

```diff
@@ -6,13 +6,38 @@ import {useIntentLink} from './useIntentLink'
  * @public
  */
 export interface IntentLinkProps {
+  /**
+   * The name of the intent to link to.
+   */
   intent: string
+
+  /**
+   * The parameters to include in the intent.
```

Comment:
Think you mentioned this earlier, but forgot the answer: Will there be a link to IntentParameters here? If not maybe we could enumerate the possible options or add a `@see IntentParameters` (or smth)

Reply:
Binoy will know the answer better I think but I think the `@link` could work for what you're asking @bjoerge